spikeinterface/comparison/multitemplatecomparison.py

```diff
@@ -0,0 +1,65 @@
+from .basecomparison import BaseComparison
+from .basemulticomparison import BaseMultiComparison
+from .templatecomparison import TemplateComparison
+
+
+class MultiTemplateComparison(BaseMultiComparison, BaseComparison):
```

Comment:
Same here I don't think we should have this cyclic stuff

Reply:
Why is this cyclic?